### PR TITLE
Update bosh and concourse

### PR DESCRIPTION
**Pull request by rogeruiz**
This patch series touches upon outdated credentials and secrets for govcloud.
This also tackles 18F/cg-atlas#62.

Secrets, etc, uploaded to govcloud as well.

### CODE REVIEW

File: bosh/jobs.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  - name: newrelic-monitor
+    release: newrelic
+  - name: collectd
+    release: collectd
+  hardening_templates: (( merge ))
+
+jobs:
+- name: docker-compute
+  templates: (( meta.docker_templates meta.hardening_templates ))
+  instances: (( merge ))
```
Comment (rogeruiz) [inline bosh/jobs.yml:14]: @jmcarp These `instances: <X>` properties are set to a number from the secrets. With a number of instances set to `instances: 0`, it shouldn't need any changes to `deployments.yml`.

**Comment by jmcarp** [comment]
Does this mean we'd use zero `docker-compute`, `docker-compute-reserve`, and `docker-highmem` instances on govcloud production? Is that going to degrade performance relative to east/west? Also, it looks like we're still hard-coding a fixed number of node IPs--does that fit with dynamic `instance` counts? https://github.com/18F/cg-deploy-docker-swarm/blob/e7ad72df9fd23fa13a943505a7f5e253e262b32f/bosh/jobs.yml#L66-L79

**Comment by rogeruiz** [comment]
I'd defer to @LinuxBozo on the node IPs, I think it would fit within it. It shouldn't degrade performance on `e/w` since that's a completely separate set of secrets which have the instance counts set there to what they were in `jobs-production.yml`.

cc @jmcarp

**Comment by LinuxBozo** [comment]
@jmcarp @rogeruiz I have no clue how the swarm manager will react to having all those hosts defined but not available when running 0 instances. to be safe, you could move those to secrets and merge as well, only defining what you have in use via instances.

**Comment by jmcarp** [comment]
@rogeruiz: I know we're not going to degrade performance _on_ east/west--I'm asking if we're going to grade performance on govcloud _relative_ to east-west. And agreed on moving hosts to secrets.

**Comment by LinuxBozo** [comment]
@jmcarp also, I think the intention here is to keep costs low on govcloud, so yes, we're making a intentional decision to not run every type of instance. Also, since there are relatively few apps on the govcloud side, and none that use the docker brokered services as of yet, this shouldn't be an issue.. as we move tenants over, we'll have to revisit.

**Comment by jmcarp** [comment]
Sounds good. Hopefully we'll be able to safely reschedule containers across hosts by the time we have tenants 😄

**Pull request merged**